phpBB/phpbb/search/fulltext_mysql.php

```diff
@@ -828,12 +827,12 @@ public function author_search($type, $firstpost_only, $sort_by_sql, $sort_key, $
 		extract($this->phpbb_dispatcher->trigger_event('core.search_mysql_author_query_before', compact($vars)));
 
 		// If the cache was completely empty count the results
-		$calc_results = ($result_count) ? '' : 'SQL_CALC_FOUND_ROWS ';
+		$sql_select	= ($type == 'posts') ? 'p.post_id' : 't.topic_id';
```

Comment:
you could remove this variable to simplify the code (like before)

Reply:
Well, it's used later for wrapping into `COUNT()`, so if I remove it here that would be the trick.